### PR TITLE
standardised util functions comments

### LINKED ISSUE #1396: Standardise Util Functions Comments
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open-source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### What is to be done?
We are working towards standardizing all the comments in the codebase to make the docs look professional. New developers will get benefitted if we add comments to some of the most commonly used util functions. They are listed below.
1. [pixelSetter.js](https://github.com/publiclab/image-sequencer/blob/main/src/util/pixelSetter.js)
2. [getDefaults.js](https://github.com/publiclab/image-sequencer/blob/main/src/util/getDefaults.js)

You will have to update the comments in the above files based on the guidelines mentioned below.
1. Methods must be described using [JSDoc comments](https://devdocs.io/jsdoc/)
2. Misc code comments should be inline unless it is a long sentence.
3. No use of continuous tenses, no pronouns.
4. No redundant comments.
5. Each comment should start with an uppercase letter and end with a full stop.

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 Modify the file [pixelSetter.js](https://github.com/publiclab/image-sequencer/blob/main/src/util/pixelSetter.js)

- [ ] 📝 Modify the file [getDefaults.js](https://github.com/publiclab/image-sequencer/blob/main/src/util/getDefaults.js)

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need it if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by tanish14**
Fixes #1396  (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm run test-all`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below
* [x] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

File: src/util/pixelSetter.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/** Sets the pixels from 0 through length of value. */
```
Comment (HarshKhandeparkar) [inline src/util/pixelSetter.js:1]: Is it possible to document these parameters `x`,`y`,`value` etc using JSDoc?
https://jsdoc.app

Comment (tanish14) [reply src/util/pixelSetter.js]: @HarshKhandeparkar Kindly review the changes, do I need to add something else? :)

File: src/util/pixelSetter.js [hunk 2]
```diff
@@ -1,5 +1,12 @@
+/**  
+ * @param {number} x x-coordinate.
+ * @param {number} y y-coordinate.
+ * @param {object} value
```
Comment (HarshKhandeparkar) [inline src/util/pixelSetter.js:4]: value is an array, `[r, g, b, a]` :)

File: src/util/pixelSetter.js [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+ * @param {number} pixel
```
Comment (HarshKhandeparkar) [inline src/util/pixelSetter.js:5]: This is not a number. You can write this:
```js
@param {object} pixels NDarray of pixels.
```

**Review by HarshKhandeparkar — CHANGES_REQUESTED** [review]

File: src/util/pixelSetter.js (hunk 2 above)
Comment (tanish14) [reply src/util/pixelSetter.js]: @HarshKhandeparkar ,made the changes. Thanks :)

File: src/util/pixelSetter.js [hunk 4]
```diff
@@ -1,5 +1,12 @@
+/**  
+ * @param {number} x x-coordinate.
+ * @param {number} y y-coordinate.
+ * @param {object} value array [r, g, b, a]
+ * param {object} pixels NDarray of pixels.
```
Comment (HarshKhandeparkar) [inline src/util/pixelSetter.js:5]: Oops, you missed a ^`@`^

Comment (tanish14) [reply src/util/pixelSetter.js]: @HarshKhandeparkar Thanks a lot for making my fist PR so easy-going and helping throughout the process :))

**Review by HarshKhandeparkar — APPROVED** [review]

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Great work, thank you!!

**Pull request merged**